lib/gtfs/cache_file.ex

```diff
@@ -0,0 +1,127 @@
+defmodule Gtfs.CacheFile do
+  @moduledoc """
+  This module holds the logic for loading and saving cache files
+  in dev and test environments with the goal of decreasing startup
+  time so devs can work more quickly.
+
+  If GTFS data changes, the cache files should be deleted/removed
+  to allowing caching of the changes.
+  """
+  require Logger
+
+  alias Gtfs.Helpers.EnvHelper
+
+  @directory Path.join([File.cwd!(), "priv/gtfs_cache"])
+
+  @dev_filename "dev_cache.terms"
+  @test_filename "test_cache.terms"
+
+  @doc """
+  The application should use the file to load state for GTFS
+  if the the envs are :dev or :test.
+
+    iex> Mix.env
+    :test
+    iex> CacheFile.should_use_file?
+    true
+  """
+  @spec should_use_file?() :: boolean
+  def should_use_file? do
+    EnvHelper.is_env?(:dev) || EnvHelper.is_env?(:test)
```

Comment:
Rather than checking this, it should be configured in `config/*.exs`. Something like:
```elixir
# config/config.exs
config :skate, Gtfs.CacheFile, use_cache?: false

# config/dev.ex
config :skate, Gtfs.CacheFile, use_cache?: true, cache_filename: "dev_cache.terms"

# config/test.ex
config :skate, Gtfs.CacheFile, use_cache?: true, cache_filename: "test_cache.terms"

def should_use? do
  Application.get_env(:skate, __MODULE__)[:use_cache?]
end

def cache_filename do
  Application.get_env(:skate, __MODULE__)[:cache_filename]
end
```

Reply:
Could `cache_filename` control whether to use the cache by whether or not it exists, instead of having a separate `use_cache?`?

Reply:
Combined both suggestions.